app/tests/e2e/helpers.js

```diff
@@ -76,7 +80,7 @@ async function waitForText(
 }
 
 async function getLastActivityItemHash(browser) {
-  await browser.waitForElementVisible(".tx-container .tx", 10000)
+  await browser.waitForElementVisible(".tx-container .tx", 50000)
```

Comment:
Maybe my timeout is too high or is it OK?

Reply:
Well, check passed, so this looks good

Reply:
I forgot this. The timout might be too high as the global timeout is only 30000